src/utils.ts

```diff
@@ -1372,13 +1442,14 @@ export class newShape {
 
   public setDrawingProperties(context: CanvasRenderingContext2D) {
     context.lineWidth = this.lineWidth;
-    context.strokeStyle = this.strokeStyle;
     context.setLineDash(this.dashLine);
-    context.globalAlpha = this.alpha;
+    if (this.alpha == 0) this.isFilled = false
+    else if (this.alpha != 1) context.globalAlpha = this.alpha;
```

Comment:
Just out of curiosity, what does happen when alpha is 1 ? Just default behavior ?

Reply:
Yes, it is to avoid to set an alpha = 1 for perf reasons